.github/workflows/publish_postgres.yml

```diff
@@ -0,0 +1,27 @@
+name: Postgres image build and upload
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+    - 'postgres/**'
+    - '.github/workflows/publish_postgres.yml'
+    - '.github/actions/build-and-push-to-quay/**'
+
+jobs:
+  push_to_registry:
+    name: Build and push image
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+      - name: Build and upload Postgres
+        uses: ./.github/actions/build-and-push-to-quay
+        with:
+          image_name: postgres
+          publish_latest: true
+          publish_calver: true
+          docker_directory: ./postgres
+          quay_user: ${{ secrets.QUAY_USER }}
+          quay_password: ${{ secrets.QUAY_PASSWORD }}
```

Comment:
This is what it will look like in a workflow. I had this execute on this branch here to make sure it's working before merge: https://github.com/CoreDB-io/coredb/actions/runs/4000451092/jobs/6865589938

Quay:  https://quay.io/repository/coredb/postgres?tab=tags